pages/affinity-diagramming.md

```diff
@@ -11,7 +11,7 @@ A way of finding themes in collections of ideas, quotes, or observations.
 
 ## Reasons to use it
 
-Quick and collaborative approach for drawing insights from qualitative data.
+To draw out insights from qualitative data quickly and collaboratively.
```

Comment:
Do we need to put this in a separate PR so it makes it into the printed version? Is it referencing an issue? (Good catch, though!)